lib/senders/templates/verification_reminder_first.txt

```diff
@@ -0,0 +1,9 @@
+{{t "Hello again."}}
+
+{{t "A few days ago you created a Firefox Account, but never verified it. A verified account lets you access your tabs, bookmarks, passwords and history on any device connected to it. Simply confirm this email address to activate your account."}}
```

Comment:
Please apply the same changes to the txt only version as I proposed to the HTML version.